src/AirshipBindings.iOS.Core/ApiDefinitions.cs

```diff
@@ -1783,35 +1848,48 @@ interface UAJavaScriptEnvironment
         string Build ();
     }
 
+    // @interface UALocaleManager : NSObject /** * The currentLocale. Setting the currentLocale to nil reverts the locale to the current user’s chosen system locale. */ @property (nonatomic, strong, nullable) NSLocale *currentLocale; /** * Clear the locale. Reverts the locale to the current user’s chosen system locale. */ - (void)clearLocale; @end
```

Comment:
I thought you had fixed these weird long comments, but I can't remember what the fix was.

Reply:
Unfortunately these are coming from sourcekitten, so there's not a ton we can do unless we want to try to get clever with text processing

Reply:
Let's not get clever.